src/Filter/ModelAutocompleteFilter.php

```diff
@@ -67,52 +70,59 @@ public function getRenderSettings()
      *
      * @param ProxyQueryInterface|QueryBuilder $queryBuilder
      * @param string                           $alias
-     * @param mixed                            $data
+     * @param mixed[]                          $value
      *
-     * @return mixed
+     * @return void
      */
-    protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $data)
+    protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $value)
     {
-        if (0 === \count($data['value'])) {
+        if (0 === \count($value['value'])) {
             return;
         }
 
         $parameterName = $this->getNewParameterName($queryBuilder);
 
-        if (isset($data['type']) && EqualOperatorType::TYPE_NOT_EQUAL === $data['type']) {
+        if (isset($value['type']) && EqualOperatorType::TYPE_NOT_EQUAL === $value['type']) {
             $this->applyWhere($queryBuilder, $queryBuilder->expr()->notIn($alias, ':'.$parameterName));
         } else {
             $this->applyWhere($queryBuilder, $queryBuilder->expr()->in($alias, ':'.$parameterName));
         }
 
-        $queryBuilder->setParameter($parameterName, $data['value']);
+        $queryBuilder->setParameter($parameterName, $value['value']);
     }
 
     /**
      * @param ProxyQueryInterface|QueryBuilder $queryBuilder
      * @param string                           $alias
-     * @param mixed                            $data
+     * @param mixed[]                          $value
      *
-     * @return mixed
+     * @return void
      */
-    protected function handleModel(ProxyQueryInterface $queryBuilder, $alias, $data)
+    protected function handleModel(ProxyQueryInterface $queryBuilder, $alias, $value)
     {
-        if (empty($data['value'])) {
+        if (empty($value['value'])) {
             return;
         }
 
         $parameterName = $this->getNewParameterName($queryBuilder);
 
-        if (isset($data['type']) && EqualOperatorType::TYPE_NOT_EQUAL === $data['type']) {
+        if (isset($value['type']) && EqualOperatorType::TYPE_NOT_EQUAL === $value['type']) {
             $this->applyWhere($queryBuilder, sprintf('%s != :%s', $alias, $parameterName));
         } else {
             $this->applyWhere($queryBuilder, sprintf('%s = :%s', $alias, $parameterName));
         }
 
-        $queryBuilder->setParameter($parameterName, $data['value']);
+        $queryBuilder->setParameter($parameterName, $value['value']);
     }
 
-    protected function association(ProxyQueryInterface $queryBuilder, $data)
+    /**
+     * @param mixed[] $value
+     *
+     * @return array
+     *
+     * @phpstan-return array(string, bool)
```

Comment:
I don't know this syntax, can't say if it's correct.

Reply:
Did you mean to write this:
```suggestion
     * @phpstan-return array{string, bool}
```
?